runtime/tests/jvmtitests/src/com/ibm/jvmti/tests/eventClassFileLoadHook/ecflh001.c

```diff
@@ -186,7 +186,7 @@ ecflh001(agentEnv * agent_env, char * args)
 	} 
 
 	/* Disable the JVMTI_EVENT_CLASS_FILE_LOAD_HOOK callback */
-	if (NULL == testArgs) {
+	if (-1 == argSet.modType) {
```

Comment:
Please update the comment for this function that suggests the `<modtype>` portion of the arguments is required.

Reply:
`ModType` should define a literal like `MOD_TYPE_UNSPECIFIED = -1` to avoid mixing `int` and `ModType` in assignments and comparisons.

Reply:
Updated, but untested. If you're happy with the changes we can start new PR testing.